riotfile.py

```diff
@@ -427,6 +427,9 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="internal",
+            env={
+                "DD_TRACE_AGENT_URL": "http://localhost:8126",
+            },
```

Comment:
```suggestion
```
I believe this is the default, so it's not necessary to make it explicit here.

Reply:
It's actually not set as a default, so I had to specify for this test suite. Would it make sense to set it as a default?